recruitment/introduction.md

```diff
@@ -0,0 +1,26 @@
+## Holiday Extras
+
+#### Why a technical task?
+
+You might be wondering why we feel its important to test candidates. At Holiday Extras we use a variety of different technologies and we don't expect you to be aware of all of them! We've designed both of our tasks with this in mind.
+
+Below is a few points that we tend to look for when reviewing your submission:
+
+- Quality (is it functional?)
+- Structure (code layout, abstraction, modularisation)
+- Usability and design
+
+#### The tasks
+
+- [Build an API](developer-API-task.md)
+- [Flickr API](developer-flickr-task.md)
+
+If you have _not_ been asked to complete a specific task in the email you received from our recruitment team then we're leaving it up to you! Feel free to pick whichever task you believe would best showcase your skills.
```

Comment:
@StefKennett I was trying to do a catch all in case SB are interviewing for a backend role in particular. They might want to specify that the candidate does the API test. Or maybe we could argue it's obvious?

@kevinhodges @notABluesSinger would be good to get your opinion!